zzz_generated/app-common/app-common/zap-generated/cluster-enums.h

```diff
@@ -1474,11 +1474,12 @@ enum class LockDataTypeEnum : uint8_t
     kRfid            = 0x07,
     kFingerprint     = 0x08,
     kFingerVein      = 0x09,
+    kFace            = 0x0A,
     // All received enum values that are not listed above will be mapped
     // to kUnknownEnumValue. This is a helper enum value that should only
     // be used by code to process how it handles receiving and unknown
     // enum value. This specific should never be transmitted.
-    kUnknownEnumValue = 10,
+    kUnknownEnumValue = 11,
```

Comment:
why not 0x0B to be consistent?

Reply:
Because This is generated code and the generated outputs decimal, and changing that is out of scope for this PR.